cryptoki/src/object.rs

```diff
@@ -254,6 +255,7 @@ impl AttributeType {
             CKA_UNWRAP_TEMPLATE => String::from(stringify!(CKA_UNWRAP_TEMPLATE)),
             CKA_DERIVE_TEMPLATE => String::from(stringify!(CKA_DERIVE_TEMPLATE)),
             CKA_ALLOWED_MECHANISMS => String::from(stringify!(CKA_ALLOWED_MECHANISMS)),
+            CKA_VENDOR_DEFINED => String::from(stringify!(CKA_VENDOR_DEFINED)),
```

Comment:
Since vendor defined attributes can go from `CKA_VENDOR_DEFINED` to the last value of `CK_ULONG`, maybe we can specify the range here and put in the string what the value is?